### PR TITLE
Calendar rebase (4) 

### PR DESCRIPTION
**Pull request by fnagel**
This replaces #1352.

### CODE REVIEW

**Comment by scottgonzalez** [comment]
@jzaefferer Do you want to go through your list again and make sure everything was either addressed or can wait and is on the todo list?

**Comment by jzaefferer** [comment]
I went through my list above and added checkboxes. I think all the regressions are addressed and this PR can land. I've merged the remaining items into the todo list on the wiki.

File: demos/calendar/buttonbar.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	<script src="../../ui/widget.js"></script>
+	<script src="../../ui/button.js"></script>
+	<script src="../../ui/calendar.js"></script>
+	<script src="../../ui/position.js"></script>
+	<link rel="stylesheet" href="../demos.css">
+	<script>
+	$(function() {
+		$( "#calendar" ).calendar({
+			buttons: {
+				"Today": function() {
```
Comment (scottgonzalez) [inline demos/calendar/buttonbar.html]: Drop the quotes.

File: demos/calendar/index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<ul>
+	<li><a href="default.html">Default functionality</a></li>
+	<li><a href="buttonbar.html">Display button bar</a></li>
+	<li><a href="dropdown-month-year.html">Display month &amp; year menus</a></li>
+	<li><a href="localization.html">Localize calendar</a></li>
+	<li><a href="min-max.html">Restrict date range</a></li>
+	<li><a href="multiple-months.html">Display multiple months</a></li>
+	<li><a href="other-months.html">Dates in other months</a></li>
+	<li><a href="show-week.html">Show week of the year</a></li>
+</ul>
```
Comment (scottgonzalez) [inline demos/calendar/index.html]: `multiple-calendars.html` isn't listed, but it looks like it's just a duplicate of `multiple-months.html`, so we should delete that file.

File: demos/calendar/localization.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	<link rel="stylesheet" href="../demos.css">
+	<script>
+	$(function() {
+		var calendar = $( "#calendar" ),
+			select = $( "#locale" );
+
+		Globalize.locale( select.val() );
+		calendar.calendar();
+
+		select.change( function() {
```
Comment (scottgonzalez) [inline demos/calendar/localization.html]: extraneous space

File: demos/calendar/localization.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+<body>
+
+<div id="calendar"></div>
+<select id="locale">
+	<option value="de-DE" selected>German (Deutsch)</option>
+	<option value="en">English</option>
+</select>
+
+<div class="demo-description">
+<p>Localize the calendar calendar language and format (English / Western formatting is the default). The calendar includes built-in support for languages that read right-to-left, such as Arabic and Hebrew.</p>
```
Comment (scottgonzalez) [inline demos/calendar/localization.html]: "calendar calendar"

File: demos/calendar/min-max.html [hunk 5]
```diff
@@ -11,23 +11,28 @@
...
 	</script>
 </head>
 <body>
 
-<p>Date: <input type="text" id="datepicker"></p>
+<div id="calendar"></div>
 
 <div class="demo-description">
-<p>Display a button for selecting Today's date and a Done button for closing the calendar with the boolean <code>showButtonPanel</code> option.  Each button is enabled by default when the bar is displayed, but can be turned off with additional options.  Button text is customizable.</p>
+<p>Restrict the range of selectable dates with the <code>min</code> and <code>max</code> options. Set the beginning and end dates as actual dates (new Date(2009, 1 - 1, 26)).</p>
```
Comment (scottgonzalez) [inline demos/calendar/min-max.html]: Put the example at the end in `<code>`.

**Comment by scottgonzalez** [comment]
A few minor comments, they can just be addressed in a new commit if you want. I think we're good to merge now. We can do any further cleanup after the merge.

File: demos/calendar/buttonbar.html (hunk 1 above)
Comment (fnagel) [inline demos/calendar/buttonbar.html]: Done.

File: demos/calendar/index.html (hunk 2 above)
Comment (fnagel) [inline demos/calendar/index.html]: Done.

File: demos/calendar/localization.html (hunk 3 above)
Comment (fnagel) [inline demos/calendar/localization.html]: Done.

File: demos/calendar/localization.html (hunk 4 above)
Comment (fnagel) [inline demos/calendar/localization.html]: Done.

File: demos/calendar/min-max.html (hunk 5 above)
Comment (fnagel) [inline demos/calendar/min-max.html]: Done.

**Comment by fnagel** [comment]
@scottgonzalez I've cleaned up the demos. Happy merging :-D

**Pull request merged**